includes/Experiments.php

```diff
@@ -219,6 +219,17 @@ public function get_experiment_groups() {
 	 */
 	public function get_experiments() {
 		return [
+			/**
+			 * Author: @samwhale
+			 * Issue: 6153
+			 * Creation date: 2021-06-07
+			 */
+			[
+				'name'        => 'enableRightClickMenus',
+				'label'       => __( 'Enable Right Click Menus', 'web-stories' ),
+				'description' => __( 'Adds a contextual shortcut menu when right clicking in the editor.', 'web-stories' ),
```

Comment:
just mentioning this here so we can keep it in the back of our brains for this feature, we need to make sure "right click" on a keyboard also works! (Shift + F10)

Reply:
GOOD CALL

Reply:
@BrittanyIRL said in a chat offline that we won't be opening the right click menu with the keyboard. The functionality of the shortcuts are bound to keys already.

Reply:
@swissspidy we can circle back to this, but it currently seems like all right click menu functionality is available in the keyboard shortcuts.